gadgets/trace_exec/gadget.yaml

```
@@ -24,6 +24,21 @@ datasources:
           columns.width: 64
           columns.hidden: "true"
           columns.alignment: left
+      dev_major:
+        annotations:
+          description: The major number of the device where the executable file is located
```

For the 3 new fields:
I wonder how to make it clearer that it's the dev major/minor of the `file` field and not the `exepath` field, in a concise way.

```suggestion
          description: The major number of the device where the executable file is located. In case of script, this refers to the script and not the interpreter
```

Then, gadgets/trace_exec/README.mdx needs to be updated.
`make -C gadgets/ trace_exec/README.mdx` could help ([gomplate](https://docs.gomplate.ca/installing/) needs to be installed)